docker-compose.yml

```diff
@@ -27,16 +24,14 @@ services:
       - net.ipv6.conf.all.forwarding=1
     networks:
       tailnord-net:
-        ipv4_address: 10.1.1.2
+        ipv4_address: ${SUBNET_PREFIX}.2
 
   nordvpn:
     build:
       context: nordvpn
-    container_name: tailnord-nordvpn
-    hostname: tailnord-nordvpn
-    entrypoint: [
-      '/scripts/nordvpn_up.sh'
-    ]
+    container_name: ${HOSTNAME}-nordvpn
```

Comment:
This will end up being something like `tailnord-tailscale-nordvpn`. Perhaps you need to instead docker compose top-level name env variable instead.